macaroonbakery/tests/test_macaroon.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+from unittest import TestCase
+
+import nacl.utils
+import base64
+
+from macaroonbakery import bakery, macaroon, checkers, codec
+
+
+class TestMacaroon(TestCase):
+    def test_new_macaroon(self):
+        m = macaroon.Macaroon(b'rootkey',
+                              b'some id',
+                              'here',
+                              bakery.LATEST_BAKERY_VERSION)
+        assert m is not None
```

Comment:
self.assertIsNone?
In general, why not using the bultin TestCase methods, that provide a better output?